app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -10,6 +10,39 @@ export const NOTES_FORM_VALIDATION_SCHEMA = yup.object().shape({
   description: yup.string().required("Description is required"),
```

Comment:
So where exactly are you validating to ensure that there is alteast one contact selected and atleast one tag selected?